src/Avalonia.Base/PixelSize.cs

```diff
@@ -77,6 +78,27 @@ public static PixelSize Parse(string s)
             }
         }
 
+        /// <summary>
+        /// Try parsing <paramref name="source"/> as <see cref="PixelSize"/>.
+        /// </summary>
+        /// <param name="source">The <see cref="string"/> to parse.</param>
+        /// <param name="result">The result of parsing.</param>
+        /// <param name="separator">Optional parameter, identifies the separator the width and height, the default value is ','</param>
+        /// <returns><c>true</c> if <paramref name="source"/> is valid <see cref="PixelSize"/>, otherwise <c>false</c>.</returns>
+        public static bool TryParse(string source, [MaybeNullWhen(false)] out PixelSize? result, char separator = ',')
```

Comment:
`MaybeNullWhen(false)` doesn't apply here: it's already nullable.

Anyways, the method should return `PixelSize` instead of `PixelSize?`, for consistency with other `TryParse` methods on structs (see `Int32`, `DateTime`, etc.)

Reply:
`PixelSize` has been changed, but `MaybeNullWhen(false)` still doesn't have any effect on a struct - its purpose is for reference types. Can you please remove it?